app/serializers/node_serializer_data.rb

```diff
@@ -457,11 +549,93 @@ module NodeSerializerData
       }
     }.merge(FLEXIBILITY_COSTS_AND_OTHER)
 
+  # If the node belongs to the V2G presentation group then
+  # add these
+  V2G_ATTRIBUTES_AND_METHODS = {
+    technical: {
+      :electricity_output_conversion => {
+        label: 'Round trip efficiency',
+        key: :round_trip_efficiency,
+        unit: '%',
+        formatter: FORMAT_FAC_TO_PERCENT
+      }
+    }
+  }.freeze
+
+  CO2_GENERIC_ATTRIBUTES_AND_METHODS = {
+    cost: {
+      'total_initial_investment_per(:plant)' => {
```

Comment:
Maybe I misinterpret this but I thought we wanted to show CCUS costs per capacity (tonne per year / hour?) rather than per unit?

Reply:
Yes that was indeed my intention, but doesn't work at the moment. I'll write it down on the future improvement list for this table :)